src/_includes/setup/get-sdk-win.md

```diff
@@ -2,7 +2,7 @@
 
 1. Download the following installation bundle to get the latest beta release of the
 Flutter SDK (for other release channels, and older builds, see the [SDK
-archive](/sdk-archive/) page.):
+archive](/get-started/sdk-archive) page.):
```

Comment:
Why place the sdk-archive under get-started? /cc @filiph 

On www we have this under the equivalent of what would here be `/development/tools/sdk/archive`. Use that instead?

Reply:
sgtm